### ISSUE TITLE
Split some Scripts for easier dev/understanding

### ISSUE THREAD

**Issue by Jeremi360** [rakugoteam/VisualNovelKit#155]
I think that there should be one script - one node in most cases.
Also this will fix remove/fix problems with using groups for menu nav bar.

- [ ] *DialogueContainer/AnswerEdit*
- [ ] *DialogueContainer/DialogLabel*
- [ ] All buttons in *NavList* - I want Navlist to work/look here like [*QuickMenu*](https://github.com/rakugoteam/VisualNovelKit/blob/1.0/addons/vnkit/InGameUI/QuickMenu/QuickMenu.gd) script, 
so all buttons will use the same new class **MenuButton** similar to **QuickMenu**.
So to sum it up, **NavList** script will only be switching between screens nothing more.
scene where buttons use **QuickButton** class
- [ ] try move setup all functions from **Screens** script to related screen
